logging/logger_registry.go

```diff
@@ -82,6 +82,9 @@ func (lr *loggerRegistry) updateLoggerLevel(name string, level Level) error {
 }
 
 func (lr *loggerRegistry) updateLoggerRegistry(logConfig []LoggerPatternConfig) error {
+	for _, name := range lr.getRegisteredLoggerNames() {
+		lr.updateLoggerLevel(name, INFO)
+	}
```

Comment:
can `lr.updateLoggerLevel(name, INFO)` just be moved into the existing loop over all registered loggers?

Reply:
I don't think so. Consider the case where you remove all logger patterns, then we never iterate through the loggers and do not reset them.